src/routes/dashboard/index.svelte

```diff
@@ -0,0 +1,417 @@
+  $: unansweredPct =
+    total && unanswered
+      ? `${Math.round((100 * parseInt(unanswered)) / parseInt(total))}%`
+      : ''
+  $: staff = filteredQuestions.get('aggregate')?.staff?.length ?? ''
+  $: staffPct =
+    total && staff
+      ? `${Math.round((100 * parseInt(staff)) / parseInt(total))}%`
+      : ''
+  $: community = filteredQuestions.get('aggregate')?.community?.length ?? ''
+  $: communityPct =
+    total && staff
+      ? `${Math.round((100 * parseInt(community)) / parseInt(total))}%`
+      : ''
+
+  $: barData = getBarData(filteredQuestions)
+  $: pieDataTotal = sortChannels(filteredQuestions.get('aggregate')!.total)
+  $: pieDataUnanswered = sortChannels(
+    filteredQuestions.get('aggregate')!.unanswered
+  )
+  $: topStaffPromise = getTopContributors(
+    filteredStaffContributors,
+    gitHubStaff,
+    9
+  )
+  $: topOverallPromise = getTopContributors(
+    filteredContributors,
+    gitHubStaff,
+    9
+  )
+</script>
+
+<svelte:head>
+  <title>Discord Metrics Dashboard</title>
+</svelte:head>
+
+<Content>
+  <Grid>
+    <Row>
+      <Column class="styled-col" style="background: rgb(15, 98, 254, 0.1);">
+        <h1 class="number">
+          {memberCount}
+          <ArrowUp size="{32}" color="#0c4fcc" />
+        </h1>
+        <h4 class="number-text">Total Members</h4>
+      </Column>
+      <Column class="styled-col" style="background: rgb(0, 255, 0, 0.1);">
+        <h1 class="number">
+          {presenceCount}
+          <Group size="{32}" color="#036b03" />
+        </h1>
+        <h4 class="number-text">Members Online</h4>
+      </Column>
+    </Row>
+    <Row class="date-container">
+      <Column style="max-width:min-content">
+        <Row>
+        <Column><h1>Questions</h1></Column><Column
+          ><Button
+            iconDescription="Download csv"
+            kind="ghost"
+            icon="{DocumentDownload}"
+            on:click={() => toCSV(channels, filteredQuestions)}
+          /></Column
+        >
+      </Row>
+      </Column>
+      <Column>
+        <FilterMenu
+          bind:dates
+          bind:channels
+          today="{today}"
+          startDate="{startDate}"
+          endDate="{endDate}"
+        />
+      </Column>
+    </Row>
+    <Row>
+      <Column class="split-counts">
+        <h1 class="number">{total}</h1>
+        <h4 class="number-text">Total Questions</h4>
+      </Column>
+      <Column class="split-counts" style="color: rgb(255, 153, 0)">
+        <h1 class="number">
+          {staff}
+          <Tag style="background-color:rgb(255, 153, 0, 0.6)">{staffPct}</Tag>
+        </h1>
+        <h4 class="number-text">Answered by Staff</h4>
+      </Column>
+      <Column class="split-counts" style="color:rgb(15, 98, 254)">
+        <h1 class="number">
+          {community}
+          <Tag style="background-color:rgb(15, 98, 254, 0.6)"
+            >{communityPct}</Tag
+          >
+        </h1>
+        <h4 class="number-text">Answered by Community</h4>
```

Comment:
we can target these headings with `h1` and `h4` in the page's `style` block rather than inline css if they're all the same

Reply:
they're actually not quite all the same :confused: